scripts/vagrant-linux-protoc.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# set up protoc so that we can use it for protobuf generation
+PROTOC_ZIP=protoc-3.3.0-linux-x86_64.zip
```

Comment:
Should we bump to 3.6.1 to get the latest and greatest? https://github.com/google/protobuf/releases/

Reply:
@schmichael sure, good catch! I'll do that and then merge.